src/app/screens/LNURLPay.tsx

```diff
@@ -110,7 +110,9 @@ function LNURLPay(props: Props) {
         }
       }
 
-      window.close();
+      if (shouldCloseWindow) {
+        window.close();
```

Comment:
if we move the window.close() into the switch case, then we do not need to set a new variable.
and it is not much difference between writing: `shouldCloseWindow = true/false` and `window.close();`

what do you think?

Reply:
What if the code doesn't pass the `(successAction && !payment.payment_error)` condition? Currently we'd close the window then too.

Reply:
👍